var/spack/repos/builtin/packages/mrcpp/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Mrcpp(CMakePackage):
+    """The MultiResolution Computation Program Package (MRCPP) is a general purpose
+    numerical mathematics library based on multiresolution analysis and the
+    multiwavelet basis which provide low-scaling algorithms as well as rigorous
+    error control in numerical computations."""
+
+    homepage = "https://mrcpp.readthedocs.io/en/latest/"
+    url = "https://github.com/MRChemSoft/mrcpp/archive/v1.1.0.tar.gz"
+
+    maintainers = ["robertodr", "stigrj", "ilfreddy"]
+
+    version('1.2.0-alpha2',
+            sha256='8f4df594751a5b7e76b09a62450c6c4956b1974876afa143cc9b5703156ccd40')
+    version('1.2.0-alpha1',
+            sha256='75797c3f6d24ec13e87e0413ecb7ff2b6caa8ed6941d58a97545331088c17f9b')
```

Comment:
I see that you're actively developing this package and that release cycles are quite short. How long do you think it will take to have `1.2.0`? I'd prefer to have that rather than 2 alpha versions if we are talking of a short time.

Reply:
We don't have an ETA on stable `1.2.0`

Reply:
Would it be fine with you to remove at least `alpha1`?

Reply:
To clarify, I'm just trying to keep alpha and beta releases in packages to the minimum possible.

Reply:
I completely understand your concern. I just let `spack create` generate the full list of releases, but there's no reason to keep multiple alphas around.